avalon/tools/models.py

```diff
@@ -393,11 +394,11 @@ def refresh(self):
 
         # Get all assets sorted by name
         db_assets = io.find({"type": "asset"}).sort("name", 1)
-        silos = db_assets.distinct("silo") or None
+        project_doc = io.find({"type": "project"})
 
-        # if any silo is set to None then it's expected it should not be used
-        if silos and None in silos:
-            silos = None
+        silos = None
+        if lib.roject_use_silo(project_doc):
```

Comment:
Typo here: `lib.roject_use_silo` should be `lib.project_use_silo`

Reply:
@iLLiCiTiT boooo